test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
`@factory` has an SRID other than 1000. rgeo 0.4 incorrectly considered two points equal if their other properties were correct but their factories were different. rgeo 0.6 correctly includes factories when determining equality. This change makes the factory on each side of the comparison equal so that the points are equal.